docs-kits/kits/Digital Twin Kit/Software Development View/page_software-development-view.md

```diff
@@ -0,0 +1,156 @@
+---
+id: Specification Digital Twin Kit
+title: Specification
+description: 'Digital Twin Kit'
+sidebar_position: 4
+---
+
+### Digital Twin Kit
+
+<!--
+Development View of the Kit.
+-->
+
+<!-- !Mandatory! -->
+## API Specifications
+
+All openAPI-specifications for the Digital Twin KIT services are rendered in the section [of these docs](API Specs)
+
+### Asset Administration Shell
+
+The Asset Administration Shell is a specification that is released by the IDTA with a perspective to be adopted bei the IEC. 
+Its mission is defining how “information about assets […] can be exchanged in a meaningful way between partners in a value
+creation network”. As such, it is well-suited to contribute to the toolbox of Catena-X. While the Spec offers an extensive
+suite of meta-model elements and APIs, Catena-X only uses a small subset. What exactly is defined in the Catena-X standard 
+CX - 0002.
+
+#### Submodels
+
+An Asset Administration Shell is organized in Submodels. Each Submodel represents a self-contained aspect of an asset - 
+typical examples are the Nameplate or AssemblyPartRelationship (which denotes the hierarchical composition of parts into
+a whole). As different aspect of an Asset may be known to different parties on the value-chain, submodels for a single asset
+must be capable to run independently of each other. The specification explicitly allows this. Catena-X demands that data-
+providers offer only a subset of the SubmodelServiceSpecification while it is advisable to expose submodels with help of 
+a full-fletched AAS-server SDK that provides the content-modifiers and API-endpoints out-of-the box.
+
+#### Digital Twin Registry
+
+What Catena-X calls the "Digital Twin Registry" (DTR) is actually the union of two different services that the AAS specification
+has specified. For simplicity's sake, they are both defined in a single service. The Registry serves a similar function as the
+index in a book: When trying to discover information, it's convenient to have an overview WHAT one will find and HOW to
+access it. The Registry caters exactly that information: For every asset it knows, it holds a number of "submodel-descriptors" and in 
+these, a consumer app will find information WHAT it will find (via the semanticId) and how to access the information (endpoint,
+security setup etcetc). As the information contained in the DTR may be sensitive and not be trusted with a central entity,
+every data provider must offer his own DTR as an EDC Data Asset.
```

Comment:
...data asset. Why data asset with capital letters?

Reply:
Again: Data Asset is a term specific to the EDC. It means something beyond what is in the word "data asset"